great-idea/index.html

```diff
@@ -14,10 +14,120 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
+<body>
+    <div class="header">
+        <nav>
+            <a href="services.html">Services</a>
+            <a href="#">Product</a>
+            <a href="#">Vision</a>
+            <a href="#">Features</a>
+            <a href="#">About</a>
+            <a href="#">Contact</a>
+            <P><a href="index.html"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a></P>
+          </nav>
+          </div>
+      
+      <div class="inandpic">
+          <h3 class="inandbut">
+            Innovation
+            <p>
+              On
+            </p>
+            <p>
+              Demand
+            </p><br></br>
+            <button>
```

Comment:
for break tags you dont have to do an open and close you can do these with just the closing or just the opening tag and it would work the same way. and for the new lines in the h3, its much better to use a break instead of adding p tags inside of it, unless you wanted to do some extra styling.(for that though you would use a span tag and not a p tag)

Reply:
Got it, thanks!